custom/icds/location_reassignment/tests/test_models.py

```diff
@@ -41,22 +41,19 @@ class TestOperation(TestCase):
 
     @classmethod
     def setUpClass(cls):
-        super(TestOperation, cls).setUpClass()
+        super(BaseTest, cls).setUpClass()
         cls.domain_obj = bootstrap_domain(cls.domain)
 
     def setUp(self):
-        super(TestOperation, self).setUp()
+        super(BaseTest, self).setUp()
         self.location_types, self.locations = setup_locations_and_types(
             self.domain,
             self.location_type_names,
             [],
             self.location_structure,
         )
 
-    def check_operation(self, old_site_codes, new_site_codes, archived=True):
-        operation = self.operation(self.domain, old_site_codes, new_site_codes)
-        operation.valid()
-        operation.perform()
+    def _validate_operation(self, operation, archived):
```

Comment:
mainly extract setup and this method to use for testing transition which internally also calls operation.